src/RoadRegistry.Product.Projections/appsettings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "BlobClientType": "FileBlobClient",
```

Comment:
Do the same as https://github.com/Informatievlaanderen/road-registry/pull/17/files#diff-f7f626b00fd553d38ca1bc95e58f48e6?

Reply:
Local dev env changes will be part of another pr. Will keep in mind.